src/java/org/jivesoftware/openfire/plugin/util/cache/ClusteredCacheFactory.java

```diff
@@ -555,7 +555,8 @@ public Condition newCondition() {
         @Override
         public V call() {
             task.run();
-            logger.debug("CallableTask[" + task.getClass().getName() + "] result: " + task.getResult());
+            if (logger.isTraceEnabled())
```

Comment:
a) `if( logger.isXXXEnabled())` is just noise provided the log statement uses templating. So this is not necessary.
b) Code style, though not strickly enforced, is for even single line `if()` statements to be surrounded by contained by curly brackets `{}`

Reply:
done